push.go

```diff
@@ -632,3 +643,37 @@ func (source *Source) GetLocaleID() string {
 	}
 	return ""
 }
+
+func (source *Source) GetFileFormat() string {
+	if source.Params != nil && source.Params.FileFormat != nil {
+		return *source.Params.FileFormat
+	}
+	if source.FileFormat != "" {
+		return source.FileFormat
+	}
+	return ""
+}
+
+func (source *Source) IncludesLocaleInformation() bool {
+	if source.Format != nil {
```

Comment:
why not just do `return source.Format != nil && source.Format.IncludesLocaleInformation`